herddb-jdbc/src/main/java/herddb/jdbc/BasicHerdDBDataSource.java

```diff
@@ -163,6 +164,14 @@ public synchronized void setUrl(String url) {
         this.url = url;
     }
 
+    public synchronized boolean isPoolConnections() {
+        return poolConnections;
+    }
+
+    public synchronized void setPoolConnections(boolean poolConnections) {
```

Comment:
IMHO to be cleaner this shouldn't be settable is client != null to not mix pooled/non pooled connection behavior (sure is a super edge and buggy case)

Reply:
done